README.md

```diff
@@ -96,16 +96,55 @@ If you're not using OpenWhisk in Bluemix or if you want to set up your Cloudant
 
 ## Listening for changes to a Cloudant database
 
+### Filter database change events
+
+You can defined a filter function, to avoid having unnecessary change events firing your trigger.
+
+To create a new filter function you can use an action.
+
+Create a json document file `design_doc.json` with the following filter function
+```json
+{
+  "doc": {
+    "_id": "_design/mailbox",
+    "filters": {
+      "by_status": "function(doc, req){if (doc.status != req.query.status){return false;} return true;}"
+    }
+  }
+}
+```
+
+Create a new design document on the database with the filter function
+
+```
+wsk action invoke /_/myCloudant/write -p dbname testdb -p overwrite true -P design_doc.json -r
+```
+```json
+{
+    "id": "_design/mailbox",
+    "ok": true,
+    "rev": "1-5c361ed5141bc7856d4d7c24e4daddfd"
+}
+```
+
+### Create the trigger using the filter function
+
 You can use the `changes` feed to configure a service to fire a trigger on every change to your Cloudant database. The parameters are as follows:
 
 - `dbname`: Name of Cloudant database.
 - `maxTriggers`: Stop firing triggers when this limit is reached. Defaults to infinite.
+- `filter`: Filter function defined on a design document.
+- `query_params`: Optional query parameter for the filter function.
 
 
-1. Create a trigger with the `changes` feed in the package binding that you created previously. Be sure to replace `/myNamespace/myCloudant` with your package name.
+1. Create a trigger with the `changes` feed in the package binding that you created previously including `filter` and `query_params` to only fire the trigger when a document is change or modify when the status is `new`.
+Be sure to replace `/_/myCloudant` with your package name.
```

Comment:
I think I meant to say added or modified.
CouchDB will notify when there is a new doc, or updated one as long the field status = new